### PR TITLE
Metadata updated, allowed_leaf_entity_types added and many english typo

### LINKED ISSUE #79: allowed_leaf_entity_types in entity statement's constraints (forces the allowed roles of the descendants)

### PR DESCRIPTION
**Pull request by peppelinux**
## Content

- Editorials: fixed links to the specs
- many english typos
- added allowed_leaf_entity_types in the federation constraints, closes https://github.com/italia/spid-cie-oidc-docs/issues/79

# Avviso SPID 

- added allowed_leaf_entity_types in federation entity configuration/statements

- [x] Ensure your files are written following RST specs (*not MD!*)
- [x] Italian version
- [x] English version
- [x] Example files 
- [x] Ask for review

### CODE REVIEW

File: docs/en/flusso_autenticazione.rst [hunk 1]
```diff
@@ -22,16 +22,16 @@ In the following, the descriptions of the flow steps, with the numbers indicated
...
+  #. The OP authenticates the user and received the user's consent to release his attributes to the RP.
 
   #. The OP redirects the user to the URL contained in the parameter *redirect_uri* specified by the RP, passing an *Authorization Code* in the Authorization Response.
 
   #. The RP sends the *Authorization Code* received at the OP *Token Endpoint*.
 
   #. The OP *Token Endpoint* releases an **ID Token**, an **Access Token** and, if expected, a **Refresh Token**.
 
-  #. The RP receives and validates the **Access Token** and the **ID Token**. For asking the attributes that the user has authorized at the point 3, it sends a request to the OP *UserInfo Endpoint* ans uses, for the authentication, the **Access Token** contained in the HTTP Authorization header.
+  #. The RP receives and validates the **Access Token** and the **ID Token**. Then requests the user's attributes to the OP *UserInfo Endpoint* ans uses, for the authentication, the **Access Token** contained in the HTTP Authorization header.
```
Comment (fmarino-ipzs) [inline docs/en/flusso_autenticazione.rst]: typo: ans --> and

File: docs/en/metadata_oidc_op.rst [hunk 2]
```diff
@@ -45,6 +45,11 @@ The EC of an OP MUST configure a metadata of type **"openid_provider"**, that MU
   * - **Claim**
     - **Description**
     - **Supported by**
+  * - **organization_name**
+    - Vedi `OpenID.Discovery#OP_Metadata`_. Organization name, see `OIDC-FED`_.
```
Comment (fmarino-ipzs) [inline docs/en/metadata_oidc_op.rst:49]: manca riga
- |spid-icon| |cieid-icon|

File: docs/en/metadata_oidc_rp.rst [hunk 3]
```diff
@@ -44,6 +44,10 @@ The RP Metadata of type **"openid_relying_party"** MUST contain at least the fol
   * - **Claim**
     - **Description**
     - **Supported by***
+  * - **organization_name**
+    - Vedi `OpenID.Discovery#OP_Metadata`_. Organization name, see `OIDC-FED`_.
```
Comment (fmarino-ipzs) [inline docs/en/metadata_oidc_rp.rst:48]: manca riga
- |spid-icon| |cieid-icon|

File: docs/en/metadata_oidc_rp.rst [hunk 4]
```diff
@@ -44,6 +44,10 @@ The RP Metadata of type **"openid_relying_party"** MUST contain at least the fol
...
+  * - **logo_uri**
+    - URL of the entity's logo, in SVG format.
```
Comment (fmarino-ipzs) [inline docs/en/metadata_oidc_rp.rst:51]: mnca riga
- |spid-icon| |cieid-icon|

File: docs/en/userinfo_endpoint.rst [hunk 5]
```diff
@@ -20,15 +20,15 @@ The UserInfo Endpoint MUST only support the method HTTP GET :rfc:`2616` and MUST
...
+ - https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#Section-4
 
 
 .. _userinfo_response:
 
 Response
 ++++++++
 
-The response of the UserInfo Endpoint MUST specify the value "application/jwt" in the "Content-Type".
+The response of the UserInfo Endpoint MUST specify the value "application/jose" in the "Content-Type".
```
Comment (fmarino-ipzs) [inline docs/en/userinfo_endpoint.rst]: Non sono sicuro che il valore corretto è jose. Mi pareva di aver letto nelle specifiche che nel caso di nested jwt il content-type debba essere jwt. Verifichiamo.

File: docs/it/metadata_oidc_op.rst [hunk 6]
```diff
@@ -45,9 +45,14 @@ L'EC di un OP DEVE configurare un metadata di tipo **"openid_provider"** DEVE co
   * - **Claim**
     - **Descrizione**
     - **Supportato da**
+  * - **organization_name**
+    - Vedi `OpenID.Discovery#OP_Metadata`_. Nome dell'organizzazione, vedi `OIDC-FED`_.
```
Comment (fmarino-ipzs) [inline docs/it/metadata_oidc_op.rst:49]: Manca riga
- |spid-icon| |cieid-icon|

File: docs/it/metadata_oidc_rp.rst [hunk 7]
```diff
@@ -45,6 +45,11 @@ Il Metadata di tipo **"openid_relying_party"** DEVE contenere almeno i seguenti
   * - **Claim**
     - **Descrizione**
     - **Supportato da** 
+  * - **organization_name**
+    - Vedi `OpenID.Discovery#OP_Metadata`_. Nome dell'organizzazione, vedi `OIDC-FED`_.
```
Comment (fmarino-ipzs) [inline docs/it/metadata_oidc_rp.rst:49]: manca riga
- |spid-icon| |cieid-icon|

**Review by fmarino-ipzs — COMMENTED** [review]
A parte alcuni piccoli typo, la cosa principale sulla quale chiedo la vostra verifica è sul valore content-type nella userinfo response (jwt o jose)

File: docs/en/userinfo_endpoint.rst (hunk 5 above)
Comment (peppelinux) [reply docs/en/userinfo_endpoint.rst]: sarebbero equivalenti ma RFC7519 definisce il content type dei nested e registra in iana application/jwt, quindi mi torna ritornarlo a application/jwt, grazie

File: docs/en/flusso_autenticazione.rst (hunk 1 above)
Comment (peppelinux) [reply docs/en/flusso_autenticazione.rst]: https://github.com/italia/spid-cie-oidc-docs/pull/81/commits/c0a12c7add6d6b7c592f8e444fb770ed1667a0da

File: docs/en/metadata_oidc_rp.rst (hunk 3 above)
Comment (peppelinux) [reply docs/en/metadata_oidc_rp.rst:48]: https://github.com/italia/spid-cie-oidc-docs/pull/81/commits/c0a12c7add6d6b7c592f8e444fb770ed1667a0da

File: docs/en/userinfo_endpoint.rst (hunk 5 above)
Comment (peppelinux) [reply docs/en/userinfo_endpoint.rst]: https://github.com/italia/spid-cie-oidc-docs/pull/81/commits/c0a12c7add6d6b7c592f8e444fb770ed1667a0da

File: docs/it/metadata_oidc_op.rst (hunk 6 above)
Comment (peppelinux) [reply docs/it/metadata_oidc_op.rst:49]: https://github.com/italia/spid-cie-oidc-docs/pull/81/commits/c0a12c7add6d6b7c592f8e444fb770ed1667a0da

File: docs/it/metadata_oidc_rp.rst (hunk 7 above)
Comment (peppelinux) [reply docs/it/metadata_oidc_rp.rst:49]: https://github.com/italia/spid-cie-oidc-docs/pull/81/commits/c0a12c7add6d6b7c592f8e444fb770ed1667a0da

File: docs/en/metadata_oidc_op.rst (hunk 2 above)
Comment (peppelinux) [reply docs/en/metadata_oidc_op.rst:49]: https://github.com/italia/spid-cie-oidc-docs/commit/c0a12c7add6d6b7c592f8e444fb770ed1667a0da

File: docs/en/metadata_oidc_rp.rst (hunk 4 above)
Comment (peppelinux) [reply docs/en/metadata_oidc_rp.rst:51]: https://github.com/italia/spid-cie-oidc-docs/commit/c0a12c7add6d6b7c592f8e444fb770ed1667a0da

**Review by fmarino-ipzs — APPROVED** [review]

**Review by nunzionapoli — APPROVED** [review]
Approvata su indicazione si Colella

**Pull request merged**